### DISCUSSION TITLE
Another Post To Ask For The Old Feed

### DISCUSSION THREAD

**Discussion by babzcraig** [community/community#37711] [Other Feature Feedback, Questions, & Ideas]
The old feed had a lot more utility to me than this one.

I could easily see the activity of current teammates.

Now I have suggestions from people I do not follow or who I've collaborated with in the past whose recent activity are not top of mind right now.

This is basically the Instagram algorithm and it prioritizes activity and inferred relationships over actual day to day utility.

Please take it back or give us an option to opt out.

**Comment by ossanna16** [comment]
Hi there @babzcraig and welcome to our community! Thank you for the feedback!
